### DISCUSSION TITLE
CoPilot Chat button Disappeared

### DISCUSSION THREAD

**Discussion by PineappleIOnic** [community/community#54921] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hey there 👋 
I gained access to the copilot chat beta and was using it to write tests for my code, then I was gone for lunch and when I returned the chat icon had disappeared on my sidebar and now does not show up

**Comment by liusinpun** [comment]
> 

me too

**Comment by davecheney** [comment, marked as answer]
@PineappleIOnic @liusinpun please update VSCode insiders and the Chat extension. They are updated daily so its important to keep on top of the updates.

**Reply by LinqLover** [reply]
Does this rule still apply today? I encountered the same thing - it worked a few days ago, but now it does not work any longer. Maybe the extension could warn me instead of silently disappearing?
